themes/default/views/clients/home.blade.php

```diff
@@ -121,7 +105,7 @@ function removeElement(element) {
                                                 {{ $product2->price !== '0.00' && $product2->price ? $product2->price . " " . config('settings::currency_sign') : __('Free') }}
                                             </td>
                                             <td class="py-3 hidden md:table-cell">
-                                                {{ $product2->expiry_date ? date('d.m.Y', strtotime($product2->expiry_date)) : __('Never') }}
+                                                {{ $product2->expiry_date ? date('d-m-Y', strtotime($product2->expiry_date)) : __('Never') }}
```

Comment:
Use `$product2->expiry_date->toDateString()` instead of `date('d-m-Y', strtotime($product2->expiry_date))`

Reply:
fixed